package.json

```diff
@@ -12,6 +12,7 @@
   "dependencies": {
     "body-parser": "^1.10.2",
     "express": "^4.11.1",
+    "cors-anywhere": "github://jcppman/cors-anywhere#3923fb8e52f4cc966fd912cff1758990ca59e65a",
```

Comment:
will switch back to the “official" version once the patch there is accepted or similar function is provided